config/locales/subscription_authentication.yml

```diff
@@ -4,8 +4,14 @@ en:
       title: Your link has expired
       description: The links we send only work for 7 days. This is to protect your personal data.
     authenticate:
-      message: "You’ve subscribed successfully"
+      message: "You’ve subscribed to GOV.UK emails"
       description:
-        immediately: "You’ll get an email from GOV.UK each time we add or update a page about: %{title}"
-        daily: "You’ll get one email a day from GOV.UK about: %{title}"
-        weekly: "You’ll get one email a week from GOV.UK about: %{title}"
+        immediately_html: |
+          <p>You’ll get an email from GOV.UK each time we add or update a page about:</p><br/>
+          <p>%{title}</p>
```

Comment:
By putting block level HTML into these I wonder if we're actually taking something that has a pattern and structure and smushing it into this one description field. 

Perhaps you've actually got a different type of flash message here that takes 3 parameters: the action title ("You've subscribed..."), the detail ("You'll get an email...") and the subject (The list title). It feels a bit like we've created a mini view in the locale file with this approach.

Reply:
I agree it has formatting, although structure might be going a bit far here, as I think the only reason for the newline is to call attention to the list title.

That said, and despite having seen this approach elsewhere, since the content needs to be tested, but is not really suitable for use in tests.

With your suggestion, we'd still need to manually inject a couple of `<br>` tags, which we could do in the controller and would also be a bit unusual, albeit perhaps a bit simpler overall. Is that what you were thinking of?

Reply:
> With your suggestion, we'd still need to manually inject a couple of <br> tags, which we could do in the controller and would also be a bit unusual, albeit perhaps a bit simpler overall. Is that what you were thinking of?

Nah, I was thinking more like the view being along the lines of:

```
      <% capture description do %>
         <p><%= flash[:success]["detail"] %></p><br><p><%= flash[:success]["subject"] %></p>
      <% end %>
      <%= render 'govuk_publishing_components/components/success_alert', {
        message: flash[:success]["message"],
        description: description
      } %>
```

Where the view understands the flash structure and knows how to convert it rather than this being a controller concern. Bit clunky I know just feels like view is the place to deal with the HTML structure.

Reply:
Ah, I wasn't so keen on that: conceptually I'm not sure "subject" means a whole lot. Looking at the other views, there's a precedent for having one flash per use case, so I've amended the (now) second-last commit to do that. It has the advantage that we're no longer passing around the list title, which avoids another instance of [this](https://trello.com/c/XoOHgGIh/473-fix-email-alert-frontend-raising-500-errors-when-users-unsubscribe-from-lists-with-long-titles).

What do you think?

(I also added another small commit, following a product review.)